.babelrc

```diff
@@ -1,5 +1,9 @@
 {
-  "presets": ["react", "es2015", "stage-0"],
+  "presets": [
+    "react",
+    ["latest", {"es2015": {"modules": false}}],
+    "stage-0"
+  ],
```

Comment:
Oh no, I see that [it defaults to the same as latest](http://babeljs.io/docs/plugins/preset-env/#how-it-works-support-all-plugins-in-babel-that-are-considered-latest). I'll switch to env!